PanCardView/Common/CardsView.cs

```diff
@@ -1720,7 +1711,7 @@ private void SendChildToBackIfNeeded(View view, View topView)
 
             if (currentIndex < backIndex)
             {
-                ExecutePreventException(() => LowerChild(view));
+                ExecutePreventException(() => view.ZIndex--);
```

Comment:
That's doesn't work really.
Do you think we can assume it's a MAUI bug?

Reply:
dunno) Didn't think up of any substitution, maybe you have any ideas?

Reply:
Nooo... dude, TBH I am exhausted with all these MAUI issues.
XamarinForms were way more "stable" if we can say so :)

Reply:
I am gonna merge this PR anyway as it fixes other issues (I hope so)

Reply:
> Nooo... dude, TBH I am exhausted with all these MAUI issues. XamarinForms were way more "stable" if we can say so :)

true. Well, let's wish the MAUI team, nothing more we can do))